python/orca/tutorial/NCF/tf_train_spark_dataframe.py

```diff
@@ -68,12 +70,19 @@ def model_creator(config):
                            backend=args.backend,
                            workers_per_node=args.workers_per_node)
 
+
 batch_size = 10240
 train_steps = math.ceil(train_df.count() / batch_size)
 val_steps = math.ceil(test_df.count() / batch_size)
 callbacks = [tf.keras.callbacks.TensorBoard(log_dir=os.path.join(args.model_dir, "logs"))] \
     if args.tensorboard else []
 
+if args.scheduler:
+    lr_callback = tf.keras.callbacks.LearningRateScheduler(scheduler, verbose=1)
+    callbacks.append(lr_callback)
+    with open(os.path.join(args.model_dir, 'lr_callback.pkl'), 'wb')as f:
```

Comment:
need to check if model_dir is local or remote path. If remote, first save to tmp and put to remote. Check pytorch_dataloader to do similar things.